README.adoc

```diff
@@ -1,3 +1,25 @@
+= C++ example project scanned on SonarCloud using Bitbucket Cloud pipelines
+// TODO: add SC status
+// TODO: add link to SC
```

Comment:
These links

Reply:
I wait for the repo to be mirrored on bitbucket and then I'll create all the links, checking that all is correctly linked